plugins/newsdownloader.koplugin/main.lua

```diff
@@ -20,6 +20,35 @@ local FILE_EXTENSION = ".html"
 local FEED_SOURCE_SUFFIX = "_rss_tmp.xml"
 local NEWS_DL_DIR, FEED_CONFIG_PATH
 
+
+local function deserializeXML(filename)
+    logger.dbg("NewsDownloader: File to deserialize: ", filename)
+    local f, e = io.open(filename, "r")
+    if f then
+        local xmltext = f:read("*a")
+        f:close()
+        return deserializeXMLString(xmltext)
```

Comment:
@houqp Did you check if this code works?

I did 'git pull' and got this exception all the time (no matter if it's creating init configuration or already has it):
>04/03/17-22:00:45 DEBUG NewsDownloader: File to deserialize:  ./news/feeds.xml
./luajit: plugins/newsdownloader.koplugin/main.lua:30: attempt to call global 'deserializeXMLString' (a nil value)
stack traceback:
        plugins/newsdownloader.koplugin/main.lua:30: in function 'deserializeXML'
        plugins/newsdownloader.koplugin/main.lua:130: in function 'loadConfigAndProcessFeeds'
        plugins/newsdownloader.koplugin/main.lua:77: in function 'callback'
        frontend/ui/widget/touchmenu.lua:576: in function 'action'
        frontend/ui/uimanager.lua:464: in function '_checkTasks'
        frontend/ui/uimanager.lua:613: in function 'handleInput'
        frontend/ui/uimanager.lua:707: in function 'run'
        ./reader.lua:188: in main chunk
        [C]: at 0x00404750

This is on PC, not tried yet on Android.